rating.html

```diff
@@ -1,17 +1,108 @@
 <!DOCTYPE html>
 <html>
-<body>
+<<head>
+    <!--Put  Tab Title Here-->
+    <title>Rate Movies</title>
+    <!--Set General Style Settings-->
+    <style>
 
-    <div class="displayMovie">
-        <h2> This is where the movie to choose will be inserted.</h2>
-        <body>
-        </body>
-    </div>
```

Comment:
Removing this should be fine as long as we are displaying similar text on the page for the user, which I see below

Reply:
I removed it, it should be good now.